src/selectors/machine/selectors.js

```diff
@@ -0,0 +1,4 @@
+import { find, get } from 'lodash';
+
+export const getMachineNode = (nodes, machine) =>
```

Comment:
can you please move this to `combined.js`? There is a convention for this name in other folders when the selector includes multiple entities.

Reply:
Done